subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/UserGuideSamplesRunner.groovy

```diff
@@ -364,6 +365,7 @@ Please run 'gradle docs:extractSamples' first"""
         Map envs = [:]
         String outputFile
         Transformer<String, String> outputFormatter
+        boolean ignoreExecution
```

Comment:
instead of adding an ignore flag, could we extract the `executable` from the sample and only execute things where the executable == 'gradle' (or is unset)?

Reply:
Good suggestion. I made the change. Can you have another look?